app/frontend/src/features/profile/profilePage/Overview.tsx

```diff
@@ -0,0 +1,57 @@
+import { Card, makeStyles } from "@material-ui/core";
+import React from "react";
+
+import Avatar from "../../../components/Avatar";
+import BarWithHelp from "../../../components/Bar/BarWithHelp";
+import Divider from "../../../components/Divider";
+import { CouchIcon, LocationIcon } from "../../../components/Icons";
+import IconText from "../../../components/IconText";
+import LabelAndText from "../../../components/LabelAndText";
+import { HostingStatus, MeetupStatus } from "../../../pb/api_pb";
+import { COMMUNITY_STANDING, COMMUNITY_STANDING_DESCRIPTION, LAST_ACTIVE, REFERENCES, RESPONSE_RATE, RESPONSE_TIME, VERIFICATION_SCORE, VERIFICATION_SCORE_DESCRIPTION } from "../../constants";
+import useCurrentUser from "../../userQueries/useCurrentUser";
+import { hostingStatusLabels, meetupStatusLabels } from "../constants";
+
+const useStyles = makeStyles((theme) => ({
+  card: {
+    padding: 16,
+    width: '30%',
+  },
+  grow: {
+    paddingTop: "100%",
+  },
+  info: {
+    marginTop: theme.spacing(0.5),
+  }
+}));
+
+export default function Overview() {
+  const classes = useStyles();
+  const { data: user } = useCurrentUser();
```

Comment:
If this will be reused for profile pages, better have user as a prop (also then you don't have to have all the null checks).